packages/toolkit/src/lib/react-query-service/model/useModels.ts

```diff
@@ -1,32 +1,57 @@
-import { useQuery } from "@tanstack/react-query";
+import { useInfiniteQuery } from "@tanstack/react-query";
 import { env } from "../../../server";
-import { listModelsQuery } from "../../vdp-sdk";
+import { Visibility, listModelsQuery } from "../../vdp-sdk";
 import type { Nullable } from "../../type";
 
 export function useModels({
```

Comment:
@orangecoloured Due to this behavior is different from normal useModels, we should maintain two set of queries, one is normal useModels. And another one will be `useInfiniteModels`

Reply:
Should we only have `useInfiniteModels` then? It's a paginated endpoint and feels like it doesn't make sense to have it without pagination.

Reply:
Yes we can do that before we have other use cases.

Reply:
Done.